reference/array/functions/array-diff.xml

```diff
@@ -93,18 +104,54 @@ Array
     </screen>
    </example>
   </para>
+
+  <para>
+   <example>
+    <title><function>array_diff</function> przykład z niepasującymi typami</title>
+    <para>
+     Dwa elementy są uznawane za równe wtedy i tylko wtedy, gdy
+     <literal>(string) $elem1 === (string) $elem2</literal>. To jest,
+     kiedy <link linkend="language.types.string.casting">reprezentacja łańcuchowa</link> jest taka sama.
```

Comment:
Osobiście tłumaczyłem jako "reprezentacja jako ciąg znaków" (jako że też tłumaczyłem sporo stron w reference/array/, tych bez En-Revision), ale tutaj sam nie jestem pewien. Wydaje mi się po prostu, że w tych czasach coraz rzadziej widuję tłumaczenie "string" na "łańcuch", ale zasadniczo to w ogole mało widzę tłumaczenia tego wyrazu, więc może mi się zdawać po prostu

Reply:
Zamienione na "reprezentacja jako ciąg znaków".